modules/globebrowsing/src/tileprovider/defaulttileprovider.cpp

```diff
@@ -70,6 +73,25 @@ namespace {
         // Determines if the tiles should be preprocessed before uploading to the GPU
         std::optional<bool> performPreProcessing;
 
+        struct CacheSettings {
+            // Specifies whether to use caching or not
+            std::optional<bool> enabled;
+
+            // The compression algorithm to use for cached tiles
+            std::optional<std::string> compression;
```

Comment:
Can do:
```cpp
enum class Compression {
   PNG,
   JPEG,
   LERC
};

std::optional<Compression> compression;
```

And people can pass in a string that is checked against one of the enum values

Reply:
That's not the only types of compression though. I opted for a simple string property to avoid having to explicitly add each option, but we can if you think it makes sense.

https://github.com/nasa-gibs/mrf/blob/master/doc/MUG.md#types-of-tile-compressions-supported-by-mrf

Reply:
Fixed as we discussed